cypress/e2e/rooms_admin/memberManagementViaAdminPageWithDoubleRoleAdmin.feature

```diff
@@ -0,0 +1,105 @@
+@regression_test
+@stable_test
+@prio_0_staging
+@group-E
+Feature: RoomMembers Administration - Able to see rooms and manage all the room members via rooms administration page
+
+    As a room administrator and member, I can view all rooms, see school member details, and add teacher members via the rooms administration page
+
+    Scenario Outline: Room administrator sees all rooms with members from same school with all roles and add teacher member via rooms administration page
+
+        # pre-condition: user logins
+        Given I am logged in as a '<student_1>' at '<namespace>'
+        Given I am logged in as a '<teacher_1>' at '<namespace>'
+        Given I am logged in as a '<teacher_2>' at '<namespace>'
+        Given I am logged in as a '<admin_1>' at '<namespace>'
+
+        # pre-condition: admin activates student visibility
+        Given student visibility for teachers in school management is 'enabled'
+
+        # pre-condition: teacher creating a new room in the origin school, becoming the owner
+        Given I am logged in as a '<teacher_1>' at '<namespace>'
+        Given a room named '<room_name>' exists
+
+        # owner adds student from same school
```

Comment:
this is not only for adding student but also a teacher as participant by the owner?